### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Morel-Victor**
Create Header and Searchbar in header

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
Ok for me :+1:

File: src/Components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          onMouseLeave={() => {
+            setStyle({ display: 'none' });
+          }}
+        >
+          <DropLogin style={style}>inscription</DropLogin>
+          <DropLogin style={style}>connexion</DropLogin>
+        </Row2>
+        {open ? (
+          <Overlay>
+            {' '}
```
Comment (Hacene010) [inline src/Components/Header/index.jsx]: Plz remove this :)

File: src/Components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            setStyle({ display: 'block' });
+          }}
+          onMouseLeave={() => {
+            setStyle({ display: 'none' });
+          }}
+        >
+          <DropLogin style={style}>inscription</DropLogin>
+          <DropLogin style={style}>connexion</DropLogin>
+        </Row2>
+        {open ? (
```
Comment (Hacene010) [inline src/Components/Header/index.jsx]: you can do it like this : {open && (...)}

File: src/Components/Header/style.jsx [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+      display: block;
+      opacity: 1;
+    }
+  }
+  @media (min-width: 1400px) {
+    display: none;
+  }
+`;
+
+export default {
```
Comment (Hacene010) [inline src/Components/Header/style.jsx]: How can this work ??? Did you run 'npm run prepare' to prepare husky ?

File: src/Components/SearchBar/style.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    height: 50px;
+    margin-bottom: 5px;
+  }
+`;
+
+export const Form = styled.form`
+  border: solid 2px var(--color--primary);
+  max-height: 40px;
+  min-height: 40px;
+  display: flex;
```
Comment (Hacene010) [inline src/Components/SearchBar/style.jsx:19]: Normally with 'display: flex' your items are online. So I don't see the use of 'flex-direction: row'

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Still some changes to make. See the comments

**Review by Caroline-CD — APPROVED** [review]
Ok for me :100:

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**